frontend/src/modules/editor/components/Editor.js

```diff
@@ -101,6 +107,34 @@ export class EditorBase extends React.Component<InternalProps> {
         );
     }
 
+    updateTranslationStatus = (
+        translationId: number,
+        change: string,
+        ignoreWarnings: ?boolean,
+    ) => {
+        const state = this.props;
+
+        if (!state.selectedEntity || !state.locale) {
+            return;
+        }
+
+        this.props.dispatch(history.actions.updateStatus(
+            change,
+            state.selectedEntity.pk,
+            state.locale.code,
+            state.parameters.resource,
+            state.pluralForm,
+            translationId,
+            state.nextEntity,
+            state.router,
+            ignoreWarnings,
+        ));
+    }
```

Comment:
This is very ugly, because `updateTranslationStatus` already exists in `EntityDetails` component, but I can't pass it to the `Editor`.